src/System.IO.FileSystem/tests/FileStream/ctor_str_fm_fa.cs

```diff
@@ -30,9 +30,10 @@ public void InvalidAccessThrows()
         public void InvalidFileModeAccessReadThrows()
         {
             FileMode[] invalidModes = { FileMode.Append, FileMode.Create, FileMode.CreateNew, FileMode.Truncate };
+            int i = 0;
             foreach (FileMode invalidMode in invalidModes)
             {
-                Assert.Throws<ArgumentException>(() => CreateFileStream(GetTestFilePath(), invalidMode, FileAccess.Read));
+                Assert.Throws<ArgumentException>(() => CreateFileStream(GetTestFilePath(i++), invalidMode, FileAccess.Read));
```

Comment:
Why does this one need to change? Since its an invalid argument, doesn't the file not actually get created?